dmc/customers/use_cases/create_webhook.rb

```diff
@@ -0,0 +1,10 @@
+module Customers
+  class CreateWebhook < Struct.new(:user, :remote_project)
+
+    def run!
+      account = GetAccount.new('github', user.id).run!
```

Comment:
Yes for the moment hardcoded everywhere then the remote project and project will have to carry this information, not implemented yet